projects.tex

```diff
@@ -1,36 +1,32 @@
-\section{PROJECTS}
+\section{MENTORSHIP \& PROJECTS}
 \begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}r}
-  Fluoroscopy Simulator – Medical College of Wisconsin & September 2019 – May 2020
+  FIRST Mentor – Charger Robotics Team 537 & October 2021 – Present
 \end{tabular*}
 \begin{bulletlist}
     \item{
-        Created UI mockups detailing the user workflow and consulted with UX professors and application users to
-        iterate on the design prior to starting development work on the application.
+        Taught programming concepts and problem solving skills to students, allowing them to continue learning independently
```

Comment:
"Allowing" ~> "equipping"
Just suggestion - we use allow a lot so good to mix it up where appropriate, in this case I think the switch is safe

Reply:
Resolved in a call with you